python_modules/airline-demo/airline_demo/test/test_pipelines.py

```diff
@@ -16,327 +19,37 @@
 @pytest.mark.db
 @pytest.mark.py3
 def test_pipelines():
-    now = datetime.datetime.utcnow()
-    timestamp = now.strftime('%Y_%m_%d_%H_%M_%S')
+    download_config_object = load_yaml_from_globs(
```

Comment:
we may want to give `load_yaml_from_globs` a different name if it's going to be the primary public API we promote for programmatically loading config

Reply:
ya definitely not a public api